### PR TITLE
VR edit app

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1224/HighFidelity-Beta-PR10861-f56110ce436a257396f2980f1a448daf1ca3009f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1224/HighFidelity-Beta-PR10861-f56110ce436a257396f2980f1a448daf1ca3009f.exe

File: scripts/vr-edit/vr-edit.js [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+    var APP_NAME = "VR EDIT",  // TODO: App name.
+        APP_ICON_INACTIVE = "icons/tablet-icons/edit-i.svg",  // TODO: App icons.
+        APP_ICON_ACTIVE = "icons/tablet-icons/edit-a.svg",
+        tablet,
+        button,
+        isAppActive = false,
+
+        VR_EDIT_SETTING = "io.highfidelity.isVREditing",  // Note: This constant is duplicated in utils.js.
+
+        VREDIT_MESSAGE_CHANNEL = "vredit-messages",
```
Comment (jherico) [inline scripts/vr-edit/vr-edit.js]: we should probably adopt a practice of namespacing message channels... `io.highfidelity.messages.vredit`

Comment (ctrlaltdavid) [inline scripts/vr-edit/vr-edit.js]: Will do

**Comment by ctrlaltdavid** [comment]
Replacement PR: https://github.com/highfidelity/hifi/pull/10892

**Pull request closed without merging**